### ISSUE TITLE
[3.3.5] DB/Creature: Grelin Whitebeard's gossip text

### ISSUE THREAD

**Issue by Weyrath** [TrinityCore/TrinityCore#19908]
**Description:**

NPC [Grelin Whitebeard](http://www.wowhead.com/npc=786) currently has gossip+questgiver flag set. But in WotLK he used to have a [quest greeting](https://github.com/TrinityCore/TrinityCore/issues/19756#issuecomment-309274238) that's not found in the `broadcast_text` table, which leads me to believe he should only have the questgiver flag set (confirmation from sniffs would be nice, but it's not something I can provide).

Here's the [WotLK version](https://youtu.be/2XoooE7TNYQ?t=275) compared to the [Cataclysm version](https://youtu.be/r9yRZ8_psFc?t=395).

**Branch(es):** 3.3.5

**TC rev. hash/commit:** 08d0105

**TDB version:** 335.63

**Fix**:
```sql
-- NPC 786 (Grelin Whitebeard) quest gossip text
UPDATE `creature_template` SET `gossip_menu_id`=0, `npcflag`=2 WHERE `entry`=786;
DELETE FROM `gossip_menu` WHERE `MenuID`=10893;
```

**Note:** With this change, Grelin Whitebeard will now only show the default "Greetings, $N" text until `quest_greeting` is implemented.

**Comment by tkrokli** [comment]
It would be better to wait until the `quest_greeting` table gets implemented in 3.3.5 before changing this. Also, I am fairly certain we don't delete content from `npc_text`, but only remove links/references in other tables where the text is linked incorrectly. BTW, regarding sniffs: there are no TC sniffs before patch 4.2.2 for creature 786, Grelin Whitebeard.

**Comment by Weyrath** [comment]
Thanks for checking!

I'll just remove the delete from npc_text then.

**Comment by offl** [comment]
Can be closed as dup of https://github.com/TrinityCore/TrinityCore/issues/19756.
https://github.com/TrinityCore/TrinityCore/issues/19756#issuecomment-301331271 updated.

**Comment by tkrokli** [comment]
Fair enough, hereby closed as duplicate of issue #19756 (DB/quest_greeting: Greeting quest text for several NPCs).
